MovieAppPetProject/MovieAppPetProject/Login/Login-ViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Login-ViewModel.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/05/05.
+//
+
+// LoginViewModel.swift
+import Foundation
+
+class LoginViewModel {
+
+    // MARK: - Variables
+    private let username = "Admin"
+    private let password = "DVTPassword"
+
+    // MARK: - Function
+    func login(username: String?, password: String?, completion: @escaping (Bool) -> Void) {
```

Comment:
no need for a completion. just let the function return true or false

Reply:
understood